### DISCUSSION TITLE
请问下我到底应该点哪个？

### DISCUSSION THREAD

**Discussion by decemberibo** [mikumifa/biliTickerBuy#270] [聊天专区]
![屏幕截图 2024-06-29 205905](https://github.com/mikumifa/biliTickerBuy/assets/174187603/db6df1b4-f06e-49be-9bf7-35846ecf2fc8)

**Comment by melon0918** [comment]
你直接clone 自动下依赖 不好么 挨个装那不麻烦死

**Comment by mikumifa** [comment]
不会python的话，用打包好的
